src/pretix/control/forms/filter.py

```diff
@@ -687,11 +689,63 @@ def __init__(self, *args, **kwargs):
         )
         self.fields['quota'].widget.choices = self.fields['quota'].choices
         for q in self.event.questions.all():
-            self.fields['question_{}'.format(q.pk)] = forms.CharField(
-                label=q.question,
-                required=False,
-                help_text=_('Exact matches only')
-            )
+            kwargs = {
+                "label": q.question,
+                "required": False,
+                "help_text": _('Exact matches only'),
```

Comment:
I'd recommend displaying that help text only for the field types where it makes sense. Esp. for the Multi-choice-question it's actually wrong as I can only select one choice and still find orders where this choice and some others were selected.
```suggestion
```